plugins/Hosting/js/siaMath.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+// Library for arbitrary precision in numbers
+const BigNumber = require('bignumber.js');
+// Ensure precision
+BigNumber.config({ DECIMAL_PLACES: 24 });
+BigNumber.config({ EXPONENTIAL_AT: 1e+9 });
+// Display # of decimals
+const DISPLAY_DECIMALS = 2;
+
+// Catch if BigNumber, property object, or anything else is passed in
+function interpret(thing) {
+	if (typeof thing === 'object') {
+		thing = thing.value;
+	}
+	return new BigNumber(thing).toNumber();
+}
```

Comment:
this file seems a little "fast and loose," if you get what I mean. Functions like `interpret`, `convert`, and `revert` are generic to the point that I can't easily get a sense of what they do and why. Basically, there's a lot of type information missing here. `formatSiacoin` and `formatBytes` are good because they specify the type. What are the rest for? Generic conversions to/from bignums?

Also, just a personal preference, but I think it's cleaner to define the functions at top-level and then have a compact `module.exports` at the bottom.